erikiado/jp2_online
# Task: View creation for Logout

#### What's this PR do?
Verify if Log out works correctly
#### Where should the reviewer start?
Revieweer should start in tosp_auth/test.py, because the functionality was already in the project. this pull request only checks if Log out works correctly in jp2_online/templates/base/home.html.

#### How should this be manually tested?

With your super user access to url/admin do a log in and back to the index page.
Click on logout button, and verify if redirects to the home page and your credentials are not present 

#### Any background context you want to provide?
This pull request is also to verify if I do the correct steps of the process to do code, a commit and a pull request.


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
